content/docs/start/index.md

```diff
@@ -76,3 +76,8 @@ by one in the next few pages:
   comparison across many ML experiments. Track your experiments with automatic
   versioning and checkpoint logging. Compare differences in parameters, metrics,
   code, and data. Apply, drop, roll back, resume, or share any experiment.
+
+- [**Checkpoints**](/doc/user-guide/checkpoints) let you track every training
+  epoch, return to any previous checkpoint, see when metrics converge or when
+  the learning rate needs to be adjusted, and even recover model weights,
+  parameters, and code.
```

Comment:
No longer needed in the Get Started index (if it's going to be a guide). But we can link from (and add a little text if needed) in https://dvc.org/doc/user-guide/experiment-management#checkpoints-in-source-code